webapp/.eslintrc.cjs

```diff
@@ -12,10 +12,17 @@ module.exports = {
     "plugin:react/jsx-runtime",
     "plugin:react-hooks/recommended",
     "plugin:jsdoc/recommended-typescript",
-    "plugin:prettier/recommended",
+    "plugin:prettier/recommended", // Must be the last one
```

Comment:
what you mean, in order it should be the last ?

Reply:
cf. https://github.com/prettier/eslint-plugin-prettier?tab=readme-ov-file#configuration-legacy-eslintrc